### PR TITLE
Closes #1520, closes #1264: Update to a-c 0.31.0

### LINKED ISSUE #1264: On first run with turbo mode, YouTube was missing play/pause buttons
## Steps to reproduce
- Open app
- Enable turbo mode in onboarding
- Click YouTube
- Play video

### Expected behavior
Play and pause shown

### Actual behavior
X in a box is shown instead: this glyph usually happens when web fonts are blocked.

I'm unable to reproduce after turning off turbo mode, even after installing the app. I wonder if WebView is caching the assets globally or it could have been a one-off error.

The best way to reproduce this would be to get a new device and try to open YouTube on Firefox before accessing YouTube from anywhere else. If we can't reproduce, we can close.

### Device information
* Fire TV device: 4K
* Firefox version: v3.0? v2.2?

### LINKED ISSUE #1520: Update to a-c 0.31.0
0.31.0 is needed to fix #1264

### PR DESCRIPTION
**Pull request by dnarcese**
- needed update to kotlin version 1.3.0
- change in permission requests api for playing DRM content

### Pull Request checklist

- [x] This PR includes thorough **tests** or an explanation of why it does not
- [x] This PR includes a **CHANGELOG entry** or does not need one

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt [hunk 1]
```diff
@@ -212,6 +213,9 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         requireWebRenderComponents.sessionUseCases.requestDesktopSite.invoke(enabled, session)
     }
 
+    override fun onContentPermissionRequested(session: Session, permissionRequest: PermissionRequest): Boolean =
```
Comment (psymoon) [inline app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt]: Is this for DRM?

File: app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoRepo.kt [hunk 2]
```diff
@@ -60,14 +59,14 @@ open class PocketVideoRepo(
 
     @UiThread // update backgroundUpdates.
     fun startBackgroundUpdates() {
-        backgroundUpdates?.cancel(CancellationException("Cancelling unexpectedly active Pocket update job to ensure only one is running"))
```
Comment (psymoon) [inline app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoRepo.kt]: So do we not catch any cancellation exception anymore?

File: app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt [hunk 3]
```diff
@@ -52,6 +53,7 @@ class PinnedTileAdapter(
 
     private val uiScope = CoroutineScope(Dispatchers.Main + uiLifecycleCancelJob)
 
+    @ExperimentalCoroutinesApi
```
Comment (psymoon) [inline app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt]: Do we need this for `onBindViewHolder`?

File: app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt (hunk 1 above)
Comment (dnarcese) [reply app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt]: Yes, a-c updated the API for allowing permission requests to play DRM videos.

File: app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt (hunk 3 above)
Comment (dnarcese) [reply app/src/main/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileAdapter.kt]: It's needed for line 65 `onBindCustomHomeTile(uiScope, holder, item)`

**Pull request merged**